playwright/tests/vetCenter.spec.js

```diff
@@ -0,0 +1,19 @@
+const { test, expect } = require('../utils/next-test')
+
+test.describe('vetCenter', () => {
+  test('vetCenter page renders', async ({ page }) => {
+    await page.goto('/pittsburgh-vet-center')
+    await expect(page).toHaveURL('/pittsburgh-vet-center')
```

Comment:
can you update this test to check for some text on the vet center page instead? pittsburgh is valid, but if we were searching like, `/kalamazoo-vet-center`, it would 404 but the page url would still pass the test